src/nodedef.cpp

```diff
@@ -462,9 +466,9 @@ void ContentFeatures::serialize(std::ostream &os, u16 protocol_version) const
 	collision_box.serialize(os, protocol_version);
 
 	// sound
-	serializeSimpleSoundSpec(sound_footstep, os);
-	serializeSimpleSoundSpec(sound_dig, os);
-	serializeSimpleSoundSpec(sound_dug, os);
+	serializeSimpleSoundSpec(sound_footstep, os, protocol_version);
+	serializeSimpleSoundSpec(sound_dig, os, protocol_version);
+	serializeSimpleSoundSpec(sound_dug, os, protocol_version);
```

Comment:
no need for a protocol bump, this can use the content features version too (see line 393)